data/mods/TEST_DATA/known_bad_density.json

```diff
@@ -774,7 +774,8 @@
       "rocuronium",
       "alumentum",
       "rebreather_filler_ko2",
-      "rebreather_filler_soda_lime"
+      "rebreather_filler_soda_lime",
+      "scythe_toy"
```

Comment:
Are you adding it to known bad density? No, don't do that. That's not how this works. If something has bad density you fix it, not write down that it's bad. Please do not do this.

Reply:
Alright dope, done. Thank you!